### ISSUE TITLE
Issue on Linux x64 (Segmentation fault) 

### ISSUE THREAD

**Issue by PT400C** [gdrive-org/gdrive#467]
Hey,

on my Linux x64 machine there is a critical problem with your application. Everytime running it, it just returns: `Segmentation fault`

Running gdb returns:

`This GDB was configured as "x86_64-linux-gnu".
Type "show configuration" for configuration details.
For bug reporting instructions, please see:
<http://www.gnu.org/software/gdb/bugs/>.
Find the GDB manual and other documentation resources online at:
<http://www.gnu.org/software/gdb/documentation/>.
For help, type "help".
Type "apropos word" to search for commands related to "word"...
Reading symbols from ./gdrive-linux-x64(1)...BFD: /test/lel/gdrive-linux-x64(1): invalid string offset 138752 >= 594 for section 'ion_r'
BFD: /test/lel/gdrive-linux-x64(1): .gnu.version_r invalid entry
Can't read symbols from /test/lel/gdrive-linux-x64(1): Bad value`

For detail, I use Debian 9.

**Comment by mbenlioglu** [comment]
I've created  a fresh install of Debian then run the following commands. It worked as I'd expect with no error whatsoever.
```console
bruce@debian:~/Desktop$ lsb_release -a
No LSB modules are available.
Distributor ID:	Debian
Description:	Debian GNU/Linux 9.9 (stretch)
Release:	9.9
Codename:	stretch
bruce@debian:~/Desktop$ uname -a
Linux debian 4.9.0-9-amd64 #1 SMP Debian 4.9.168-1+deb9u3 (2019-06-16) x86_64 GNU/Linux
bruce@debian:~/Desktop$ curl -fsSLo gdrive https://drive.google.com/uc?id=1Ej8VgsW5RgK66Btb9p74tSdHMH3p4UNb&export=download
bruce@debian:~/Desktop$ chmod +x gdrive 
bruce@debian:~/Desktop$ ./gdrive version
gdrive: 2.1.0
Golang: go1.10.4
OS/Arch: linux/amd64
bruce@debian:~/Desktop$
```

**Comment by mbenlioglu** [comment]
@PT400C  I couldn't replicate this error. Could you give more information?

**Comment by PT400C** [comment]
@mbenlioglu It seems that I messed up something with downloading the binaries (When I tried your commands, everything works). 
So what I did:
I chose gdrive-linux-x64 from the downloads list (https://drive.google.com/uc?id=1Ej8VgsW5RgK66Btb9p74tSdHMH3p4UNb&export=download), which you also did.

Then I downloaded it on my windows machine and uploaded it via FTP. It seems that this somehow corrupts the binary file. And idea why this happens?
